source/components/molecules/DayPicker/DayPicker.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useContext } from "react";
+import { Calendar, LocaleConfig } from "react-native-calendars";
+import moment from "moment";
+import { ThemeContext } from "styled-components/native";
+import localeConfig from "./localeConfig";
+
+LocaleConfig.locales.se = localeConfig;
+LocaleConfig.defaultLocale = "se";
+
+interface DayPickerProps {
+  availableDates: string[];
+  onDateSelected: (dateString: string) => void;
+  startDate?: string;
+}
+
+const DayPicker: React.FC<DayPickerProps> = ({
+  availableDates,
+  onDateSelected,
+  startDate,
+}) => {
+  const [selectedDate, setSelectedDate] = useState();
```

Comment:
Do we really need the selectedDate state?
I mean, if we already have a callback function called onDateSelected, we can assume that the parent rendering this calendar component also have some logic to save the selected date right?

Reply:
Good catch. I have changed the component to be stateless and instead use a `selectedDate` prop from the parent.